### PR TITLE
austin.intro.2.soape.jonathan

### PR DESCRIPTION
**Pull request by soapej**

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,56 @@
...
+            </a>
+          </td>
+          <td>
+            <a href="https://www.youtube.com/watch?v=qBh25WDiSpY">
+              <img src="http://i.dailymail.co.uk/i/pix/2012/06/12/article-2157978-136B572F000005DC-202_634x500.jpg" class="service"/>
+            </a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/in-class-project/index.html:53]: this looks fantastic! great work! don't forget the contact information here.

File: 02LessonTwo/homework/css/style.css [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+  color:white;
+  font-size:200%;
+  text-decoration:underline;
+  text-align: center;
+}
+.goals{
+  font-size:180%;
+  list-style-type: none;
+  margin-left:500px;
+}
```
Comment (ethanstenis) [inline 02LessonTwo/homework/css/style.css:262]: Wow! Amazing job!

File: 02LessonTwo/homework/index.html [hunk 3]
```diff
@@ -1,10 +1,173 @@
...
+
+<!--Put in blocks saying Austin, GT, etc
+    then give a hover effect to show another
+    image -->
+
+<!--Make sure all images have alt and title attributes-->
+
+<!--Download font pack to make more fancy-->
 
 </body>
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html:172]: Outstanding work here. You spent a lot of time and put a lot of thought into this and it shows. Keep pushing yourself and you will only get better. I'd love to see what a Ninja-Cowboy costume looks like.

File: 03LessonThree/in-class-project/css/daily-planet-style.css [hunk 4]
```diff
@@ -7,18 +7,55 @@ header, nav, main, section, aside, footer {
...
+}
+figcaption{
+  text-align:center;
+  padding-left:30px;
+  text-decoration:underline;
+  font-size:120%;
+}
+aside p{
+  margin-left:30px;
+}
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/css/daily-planet-style.css:61]: Sometimes you just have to mess with % to get the design layout you want. Your solid grasp of CSS is evident. Keep up the good work.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 5]
```diff
@@ -1,10 +1,49 @@
...
+      <p>123 Fake St. Austin, TX 78747</p>
+      <a href="https://www.facebook.com/DailyPlanetShow/?fref=nf">
+        <img src="https://www.facebookbrand.com/img/fb-art.jpg" alt="Facebook logo" title="Check out our Facebook" style="height:50px" "width:50px"/></a>
+      <a href="https://twitter.com/dailyplanetdc">
+        <img src="http://seeklogo.com/images/T/twitter-logo-4BD1CE8FF3-seeklogo.com.gif" alt="Twitter logo" title="Check out our Twitter" style="height:50px" "width:50px"/></a>
+
+    </footer>
+  </main>
+  </body>
+</html>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:49]: Nailed it. If you have time next time, use the dev tools to play around with section alignments so aside and article are same length.

File: 07LessonSeven/in-class-project/css/my-style.css [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+  margin-left:40px;
+  font-size:80px;
+}
+
+h3{
+  font-size:20px;
+}
+.text{
+  background-color:#e0e0e0;
+  margin-top:220px;
```
Comment (ethanstenis) [inline 07LessonSeven/in-class-project/css/my-style.css:27]: You could take the margin-top down a bit here, but overall this looks solid. Good job!

File: 11LessonEleven/in-class-project/index.html [hunk 7]
```diff
@@ -1,10 +1,63 @@
...
+
+      <div class="titleTxt">
+        <h2>Title</h2>
+        <p>Bacon ipsum dolor amet pig porchetta tongue tenderloin ham hock cupim strip steak sausage beef hamburger meatball ribeye. Landjaeger drumstick</p>
+      </div>
+    </div>
+
+  </section>
 </body>
 </html>
```
Comment (ethanstenis) [inline 11LessonEleven/in-class-project/index.html:63]: Looks like there is some centering issues when you transition from different media queries. Please fix and I can update your grade.
screen shot 2016-09-13 at 4 03 02 pm
screen shot 2016-09-13 at 4 02 56 pm

**Pull request closed without merging**